common/dynamicconfig/constants.go

```diff
@@ -508,6 +508,11 @@ DescribeTaskQueue API with ReportTaskQueueReachability==true, or to the GetWorke
 		10*time.Minute,
 		`ReachabilityCacheClosedWFsTTL is the TTL for the reachability closed workflows cache.`,
 	)
+	TaskQueueInternalInfoCacheTTL = NewGlobalDurationSetting(
+		"matching.wv.TaskQueueInternalInfoCacheTTL", // todo Shivam - what the heck is wv?
+		time.Minute,
```

Comment:
this seems too high, maybe let's do 1 sec?

Reply:
changed 👍🏻